ReadMe.md

```diff
@@ -1,18 +1,197 @@
-# Hello-Microverse_Project
 
-Hello-Microverse_project is a beginner level project at Microverse, that requires requires the student to display hello on a web page with simple styling. 
 
-# Built With
-HTML
-CSS
+# 📗 Table of Contents
 
-# Key Features
-1. Linter for HTML and CSS: checks errors and organise code
-2. gitignore: to keep some directories from being committed
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+
+
+# 📖 Hello-Microverse_project <a name="about-project"></a>
+
+
+**Hello-Microverse_project** is a beginner level project at Microverse. The project requires a student to add "Hello Microverse" to a webpage and apply some basic styling to it. It also requires the use of linters, .gitignore file and other best practices.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+
+<details>
+  <summary>Linters</summary>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Linters**
+- **.gitignore file**
+- ****
+
```

Comment:
Please this section should include key features of the app you have implemented. Example

- Use linter
- Use Professional documentation
- Use HTML and CSS